subtests/docker_cli/build/build.py

```diff
@@ -79,6 +87,8 @@ def setup(self):
 class build_base(subtest.SubSubtest):
 
     def dockerfile_dir_path(self, dir_path):
+        if isinstance(dir_path, basestring) or dir_path is None:
```

Comment:
The isinstance() will return true normally (I'm not try with other value, just dir_path = "/full"), is it excepted for some reseaons? Or should we change to "not isinstance()" to avoid go into this if....